include/capstone/ppc.h

```diff
@@ -66,7 +66,7 @@ typedef struct cs_ppc_op {
 	ppc_op_type type;	// operand type
 	union {
 		unsigned int reg;	// register value for REG operand
-		int32_t imm;		// immediate value for IMM operand
+		uint64_t imm;		// immediate value for IMM operand
```

Comment:
Yes, I guess `int64_t` is fine it it's in line with the rest of the code. I tend to think that `unsigned` types are easier to use for binary things (overflow is not undefined).